pyiode/reports.pyx

```diff
@@ -10,6 +10,10 @@
 
 from reports cimport B_ReportExec, B_ReportLine
 
+# GDM> I don't really know how report works, so I don't know if a report loads a workspace inside the report
```

Comment:
At the second you import the Python `iode` package (or start the new GUI), 7 global databases are created in memory. These databases are destroyed when you quit Python (or the GUI). 
The method `report_exec` below (newly called `execute_report`) will apply each IODE command contained in the IODE report. The execution of an IODE report (or an IODE command via the Python function `execute_command`) will NOT create new databases and it does NOT with a separate memory space. With IODE, everything is global (sigh).

So
> I don't know if a report loads a workspace inside the report

The answer is no.

> if you execute a report once a workspace is already loaded

Yes.

> If it is the second option, maybe make this a method on the Workspace class

An IODE report (or command) is not always about workspace. I am trying to stay more or less close to the non-Python API which provide the convenient IODE function `ReportExec`.

I also think that the users will not understand why the Python functions `execute_report` and `execute_command` are attached to a top level *Workspace* class.

For the moment, I prefer to keep the functions `execute_report` and `execute_command` as it.
In the future, if users ask for this, I would be keen to change my mind.